src/applications/disability-benefits/all-claims/containers/MissingServices.jsx

```diff
@@ -1,30 +1,70 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-export const MissingServices = ({ children }) => {
+const TelephoneLink = () => (
+  <a
+    href="tel:+18008271000"
+    aria-label="8 0 0. 8 2 7. 1 0 0 0."
+    className="no-wrap"
+  >
+    800-827-1000
+  </a>
+);
+
+const Alert = ({ content }) => (
+  <div className="vads-l-grid-container vads-u-padding-bottom--5">
+    <div className="usa-content">
+      <h1>File for Disability Compensation</h1>
+      <AlertBox isVisible content={content} status="error" />
+    </div>
+  </div>
+);
+
+export const MissingServices = () => {
   const content = (
     <>
-      For help with your application, please call Veterans Benefits Assistance
-      at{' '}
-      <a
-        href="tel:+18008271000"
-        aria-label="8 0 0. 8 2 7. 1 0 0 0."
-        className="no-wrap"
-      >
-        800-827-1000
-      </a>
-      , Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
+      <h2 className="vads-u-display--inline-block vads-u-font-size--h3 vads-u-margin-top--0">
+        We’re sorry. It looks like we’re missing some information needed for
+        your application
+      </h2>
+      <p>
+        For help with your application, please call Veterans Benefits Assistance
+        at <TelephoneLink />, Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
+      </p>
     </>
   );
-  return (
-    <div className="usa-grid full-page-alert">
-      <AlertBox
-        isVisible
-        headline="We’re sorry. It looks like we’re missing some information needed for your application"
-        content={content}
-        status="error"
-      />
-      {children}
-    </div>
+  return <Alert content={content} />;
+};
+
+export const MissingId = ({ children }) => {
+  const content = (
+    <>
+      <h2 className="vads-u-display--inline-block vads-u-font-size--h3 vads-u-margin-top--0">
+        A technical error has occurred
+      </h2>
+      <p>
+        Some important technical details are missing from your account,
+        preventing you from using this tool. Please call us at <TelephoneLink />{' '}
+        so we can fix this issue for you.
+      </p>
+      <p>
+        It may expedite the process to inform the support representative that
+        you are missing either your{' '}
+        <dfn>
+          <abbr title="Electronic Data Interchange Personal Identifier">
+            EDIPI
+          </abbr>
+        </dfn>
+        {' or '}
+        <dfn>
+          <abbr title="Beneficiary Identification and Records Locator (Sub)System">
+            BIRLS ID
+          </abbr>
+        </dfn>{' '}
```

Comment:
@jenstrickland I added you to this review since I ended up with some abbreviations (which Peggy may decide aren't necessary), that I wrapped in `dfn` and `abbr` tags (above). Would you please let me know if these are appropriate?

I also added an `h1` tag and had to include an `h2` tag inside the alert since using the `headline` property was adding an `h3` messing up the order ([docs](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/))

Reply:
Nice! What was the thinking behind wrapping the `<abbr>` in a `<dfn>`?

Reply:
I just happened to have [the comment on download links](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340) opened and copied that HTML.

Reply:
Ah, in that case the `<dfn>` serves to describe the "Download the Voices of Veterans", provide the context that it is a PDF.  I suspect the `<dfn>` isn't necessary here.